contents/docs/web-analytics/web-vitals.mdx

```diff
@@ -137,7 +137,7 @@ You can choose which metrics to capture by using the toggles.
 
 ### Configuring web vitals autocapture
 
-You can provide a client side config to tune web vitals autocapture. These are set on [the `capture_performance` key](https://github.com/PostHog/posthog-js/blob/main/src/types.ts) in the `PostHogConfig` interface.
+You can provide a client side config to tune web vitals autocapture. These are set on the `capture_performance` key in the [`PostHogConfig` interface](/docs/references/posthog-js/types/PostHogConfig).
```

Comment:
Wishlist, but it would be nice to link to `capture_performance` here. Maybe set the method name as the ID of the section?